### DISCUSSION TITLE
Looking for some clarifications before using your service.

### DISCUSSION THREAD

**Discussion by PassionateG1t** [ReadYouApp/ReadYou#205] [Q&A] [answered]
Hello! Your app looks gorgeous, I was planning on usingvit but I've been doing some research and stumbled upon several things that I hope an expert can clarify for me if it's not too much to ask.

Privacy policy states that: 

> This policy does not apply to third-party services that you use with Read You.

It makes sense, but what does that mean when accessing the content from your app? Can I get tracked by those services? If so, using my hardened browser would be better privacy wise, right?

Also, README mentions as a feature:

> Parse Full Content

What does "Full Content" mean? Does it include scripts and cookies?

**Comment by Ashinch** [comment, marked as answer]
> It makes sense, but what does that mean when accessing the content from your app? Can I get tracked by those services? If so, using my hardened browser would be better privacy wise, right?

For example, if you sync data through the Fever API, your privacy should be the responsibility of the Fever API service provider.

> What does "Full Content" mean? Does it include scripts and cookies?

HTML `<content>` text only, no cookies and scripts.

**Reply by PassionateG1t** [reply]
Thank you for your clarification!
